stdlib/Printf/src/Printf.jl

```diff
@@ -726,6 +790,24 @@ const UNROLL_UPTO = 16
     return pos
 end
 
+@inline function plength(f::Spec{T}, args, argp) where {T}
+    dynamic_width, dynamic_precision = f.dynamic_width, f.dynamic_precision
+    if dynamic_width || dynamic_precision
+        width, precision = f.width, f.precision
+        if dynamic_width
+            width = args[argp]
+            argp += 1
+        end
+        if dynamic_precision
+            precision = args[argp]
+            argp += 1
+        end
+        f = Spec{T}(f.leftalign, f.plus, f.space, f.zero, f.hash, width, precision, f.dynamic_width, f.dynamic_precision)
```

Comment:
Not `false, false` for the last two arguments?

Maybe this code can be refactored to share code with `fmt` above, e.g.
```jl
f, argp = rmdynamic(f, args, argp)
```

Reply:
true,  will fix